app/components/confirm-existing-instance-modal.hbs

```diff
@@ -0,0 +1,39 @@
+<AuModal @modalOpen={{true}} @closeModal={{this.close}}>
+  <:title>Dit concept werd al gebruikt</:title>
+  <:body>
+    <div>
+      Er bestaat al (minstens) een instantie op basis van dit concept. Vermijd meerdere versies van éénzelfde product of
+      dienst. Zo
+      voorkom je uiteenlopende of verouderde informatie en extra werklast. Is de bestaande instantie niet meer relevant?
+      Verwijder deze dan.
+    </div>
+    <div class="au-u-margin-top">
+      <span class="au-u-h4 au-u-medium">Bestaande instanties</span>
+      {{#each @data.existingInstances as |instance|}}
+      <p>
+        <AuLink class="au-u-margin-right" @route="public-services.details" @model={{instance.id}} @query={{hash
+          preview=true}} {{on "click" this.close}}>
+          {{instance.nameNl}}
+        </AuLink>
+        gewijzigd op ({{moment-format instance.dateModified "DD-MM-YYYY - HH:mm"}})
```

Comment:
```suggestion
        gewijzigd op {{moment-format instance.dateModified "DD-MM-YYYY - HH:mm"}}
```

Those parentheses around the date seem a little strange to me here. Was this copied from somewhere else perhaps?

Reply:
Yeah I wasn't sure, I'll double check with Geertrui